modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ModalMoveObjectDefinition.tsx

```diff
@@ -97,7 +99,15 @@ export function ModalMoveObjectDefinition({
 				type: 'success',
 			});
 
-			setTimeout(() => window.location.reload(), 1000);
+			if (
+				Liferay.FeatureFlags['LPS-148856'] &&
```

Comment:
Hello @rebsilva, Since the move modal call was already covered by the feature flag [here](https://github.com/liferay/liferay-portal/blob/2b6e533dba0b42c5c215935188f03065a3ce744e/modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ViewObjectDefinitions.tsx#L367). Do you think we need to validate this here again?

There is already a PR in the object queue removing Model Builder FF. Depending on which PR to enter first, Paulo will need to update his PR, or you will need to update yours so that there are no inconsistencies.

Do you think it is safe to remove FF from this conditional?

Reply:
you are right @igor-franca, thanks for reminding me!